Formula/nvc.rb

```diff
@@ -1,8 +1,17 @@
 class Nvc < Formula
   desc "VHDL compiler and simulator"
   homepage "https://github.com/nickg/nvc"
-  url "https://github.com/nickg/nvc/releases/download/r1.4.0/nvc-1.4.0.tar.gz"
-  sha256 "1a874bde284408c137a93b22f8f12b5b8c3368cefe30f3a5458ccdeffa0c6ad6"
+  revision 1
+
+  stable do
+    url "https://github.com/nickg/nvc/releases/download/r1.4.0/nvc-1.4.0.tar.gz"
+    sha256 "1a874bde284408c137a93b22f8f12b5b8c3368cefe30f3a5458ccdeffa0c6ad6"
+    # Upstream issue, only fix on master branch
```

Comment:
Could you please provide a short summary of what this patch is fixing

Reply:
added in https://github.com/Homebrew/homebrew-core/pull/54741/commits/b8680430778043d5e5b2956064f0463239931064